api/xverse.ts

```diff
@@ -304,10 +307,16 @@ export async function getInscription(
   return response.data;
 }
 
-export async function getAppConfig(network: NetworkType) {
+export async function getAppConfig(network: NetworkType): Promise<AppConfig> {
   const appConfigUrl = `${XVERSE_API_BASE_URL(network)}/v1/app-config`;
-  const appConfig = await axios.get(appConfigUrl);
-  return appConfig;
+  const response = await axios.get(appConfigUrl);
+  return response.data.appConfig;
+}
+
+export async function getFeaturedDapps(sectionsEnabled = false): Promise<(FeaturedDapp | FeaturedDappOld)[]> {
```

Comment:
If `FeaturedDapp | FeaturedDappOld` is due to an api schema change, then we should be changing the end point to a v2 endpoint

Reply:
Makes sense.. we won't need the `sectionsEnabled` param then, right?